src/gof.jl

```diff
@@ -0,0 +1,168 @@
+# This file is a part of LegendSpecFits.jl, licensed under the MIT License (MIT).
+"""
+`gof.jl`
+several functions to calculate goodness-of-fit (gof) for fits (-> `specfits.jl`):
+"""
```

Comment:
```suggestion

```
We can remove that too, that docstring doesn't appear somewhere in the docs later.